flutter/lib/benchmark/state.dart

```diff
@@ -84,11 +85,15 @@ class BenchmarkState extends ChangeNotifier {
 
   late BenchmarkList _middle;
 
-  BenchmarkState._(this._store, this.backendBridge) {
+  BenchmarkState._(this._store, this.backendBridge, this.firebaseManager) {
     resourceManager = ResourceManager(notifyListeners);
     backendInfo = BackendInfo.findMatching();
   }
 
+  Future<void> testUpload() async {
```

Comment:
What does “testUpload” mean?

Reply:
I created this function when I first connected the app to firebase and it didn't function properly, and forgot to rename it when I correctly implemented the upload.
Fixed.